Fix: #48

MOHIT-IITP opened the pull request:
## Description

Fixes #48 

## Type of change
![Screenshot 2024-10-12 at 11 33 15 AM](https://github.com/user-attachments/assets/315486a0-2efe-4796-9381-ca02b90f6a4f)

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change)
- [ ] New feature (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so that they can be reproduced.

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

bryans-go reviewed (changes requested):
![image](https://github.com/user-attachments/assets/a7b00b49-8544-477c-858d-a2e8db836f05)
Please make the footer transparent and also Do not use Image as a background because it will increase the loading time of the website

MOHIT-IITP commented:
@bryans-go  changes done 
1: added css gradient and removed the bg (image)
2: footer background transparent (done)
![Screenshot 2024-10-12 at 5 39 08 PM](https://github.com/user-attachments/assets/560d08da-731c-4fff-898f-8d4dba904a1a)

MOHIT-IITP commented:
@bryans-go  i have resolve the conflicts . please check
